extension/src/popup/components/account/AssetDetail/index.tsx

```diff
@@ -216,52 +216,54 @@ export const AssetDetail = ({
           )}
         </SimpleBar>
       </div>
-      <SlideupModal isModalOpen={isModalOpen} setIsModalOpen={setIsModalOpen}>
-        <div className="AssetDetail__info-modal">
-          <div className="AssetDetail__info-modal__total-box">
-            <div className="AssetDetail__info-modal__asset-code">
-              <img src={StellarLogo} alt="Network icon" />{" "}
-              <div>{assetCode}</div>
-            </div>
-            <div>{balanceTotal}</div>
-          </div>
-          <div className="AssetDetail__info-modal__available-box">
-            <div className="AssetDetail__info-modal__balance-row">
-              <div>{t("Total Balance")}</div>
+      {isNative && (
```

Comment:
unrelated to this PR, but I realized the bottom content should only be shown native (originally it was hidden at the bottom for all non-native assets before I changed the overflow to be able to scroll in a previous PR)